server/routes/maintainingReports/coordinator.ts

```diff
@@ -13,9 +13,59 @@ export default class CoordinatorRoutes {
     private readonly involvedStaffService: InvolvedStaffService,
     private readonly reviewService: ReviewService,
     private readonly offenderService: OffenderService,
-    private readonly systemToken: SystemToken
+    private readonly systemToken: SystemToken,
+    private readonly userService: UserService
   ) {}
 
+  viewRemovalRequest: RequestHandler = async (req, res) => {
+    const { reportId, statementId } = req.params
+    const token = await this.systemToken(res.locals.user.username)
+    const [{ name, userId, email }, removalReason] = await Promise.all([
+      this.involvedStaffService.loadInvolvedStaff(parseInt(reportId, 10), parseInt(statementId, 10)),
+      this.involvedStaffService.getInvolvedStaffRemovalRequestedReason(parseInt(statementId, 10)),
+    ])
+    const location = await this.userService.getUserLocation(token, userId)
+    const data = {
+      name,
+      userId,
+      location,
+      email,
+      removalReason,
+    }
+    const errors = req.flash('errors')
+    res.render('pages/coordinator/view-removal-request.html', { data, errors })
+  }
+
+  submitRemovalRequest: RequestHandler = async (req, res) => {
+    const { reportId, statementId } = req.params
+    const { confirm } = req.body
+    const endpoint = `/coordinator/report/${reportId}/statement/${statementId}`
+
+    if (!confirm) {
+      req.flash('errors', [
+        { href: '#confirm', text: 'Select yes if you want to remove this person from the incident' },
+      ])
+      return res.redirect(`${endpoint}/view-removal-request`)
+    }
+
+    return confirm === 'yes'
+      ? res.redirect(`${endpoint}/confirm-delete`)
+      : res.redirect(`${endpoint}/staff-member-not-removed`)
+  }
+
+  viewStaffMemberNotRemoved: RequestHandler = async (req, res) => {
+    const { reportId, statementId } = req.params
+    const returnToIncidentLink = `/${reportId}/view-statements`
```

Comment:
this should just be hardcoded in the page rather than passed through.

Reply:
pass through report id to page and build url in page

Reply:
Done